nemo/nemo/utils/exp_logging.py

```diff
@@ -67,7 +69,8 @@ def __init__(
             ckpt_dir=None,
             tb_dir=None,
             files_to_copy=None,
-            add_time=True):
+            add_time=True,
+            broadcast_func=None):
```

Comment:
The same here: description of all those parameters is missing

Reply:
@chiphuyen Can you take a first stab at documenting ExpManager?

Reply:
Added a docstring

Reply:
thanks!